changes/tasks.py

```diff
@@ -572,14 +572,38 @@ def switch_channel(self, change):
                 )
 
             elif change.data["channel"] == "sms" and "whatsapp" in short_name:
-                # Change any WhatsApp subscriptions to SMS
-                sbm_client.update_subscription(sub["id"], {"active": False})
```

Comment:
This means that we'll no longer be deactivating the service info messageset, which we still want to do.